main.py

```diff
@@ -0,0 +1,51 @@
+import sys
+from PIL import Image
+from pathlib import Path
+from random import randint
+valid_images = [".jpg", ".jpeg", ".png"]
+
+
+def get_images(location):
+    images = []
+    directory = Path(location)
+    try:
+        for file in directory.iterdir():
+            ext = Path(file).suffix
+            if file.is_file() and ext in valid_images:
+                images.append(file)
+    except FileNotFoundError as e:
+        print(e)
```

Comment:
remove print of error.
relevant in all locations,
if it wasn't clear why from our talk I can explain again

Reply:
done